### PR TITLE
Include roundtrip latency and seq nr in heartbeat debug log

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala [hunk 1]
```diff
@@ -240,8 +240,14 @@ private[cluster] class ClusterHeartbeatSender extends Actor {
   }
 
   def heartbeatRsp(response: HeartbeatRsp): Unit = {
-    // TODO: log response time and validate sequence nrs once serialisation of sendTime is released
-    if (verboseHeartbeat) logDebug("Heartbeat response from [{}]", response.from.address)
+    if (verboseHeartbeat) {
+      val latencyMs = (System.nanoTime() - response.creationTimeNanos) / 1000000
+      logDebug(
+        "Heartbeat response from [{}] (round trip {} ms, seq nr [{}])",
```
Comment (patriknw) [inline akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala:246]:
```suggestion
        "Heartbeat response from [{}] (round trip [{}] ms, seq nr [{}])",
```

File: akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala [hunk 2]
```diff
@@ -240,8 +240,14 @@ private[cluster] class ClusterHeartbeatSender extends Actor {
...
+        response.from.address,
+        latencyMs,
+        response.sequenceNr)
+    }
```
Comment (patriknw) [inline akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala:250]: We should also include the seq nr in the logs of ClusterHeartbeatReceiver. Maybe prefix all those logs with something like
```
Heartbeat #$seqNr 
```

Comment (patriknw) [reply akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala:250]: and there is also one more in the sender
```
logDebug("Heartbeat to [{}]", to.address)
```

3 places in total

**Review by pvlugter — APPROVED** [review]

File: akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala [hunk 3]
```diff
@@ -212,9 +212,9 @@ private[cluster] class ClusterHeartbeatSender extends Actor {
     val nextHB = selfHeartbeat()
     state.activeReceivers.foreach { to =>
       if (failureDetector.isMonitoring(to.address)) {
-        if (verboseHeartbeat) logDebug("Heartbeat to [{}]", to.address)
+        if (verboseHeartbeat) logDebug("Heartbeat #{} to [{}]", nextHB.sequenceNr, to.address)
       } else {
-        if (verboseHeartbeat) logDebug("First Heartbeat to [{}]", to.address)
+        if (verboseHeartbeat) logDebug("First Heartbeat (#{}) to [{}]", nextHB.sequenceNr, to.address)
```
Comment (patriknw) [inline akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala:217]:
```suggestion
        if (verboseHeartbeat) logDebug("First Heartbeat #{} to [{}]", nextHB.sequenceNr, to.address)
```

Comment (patriknw) [reply akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala:217]: The ClusterHeartbeatSender has a single seqNr counter that is used when sending to all of its receivers. The receivers may change over time, so it's not always seqNr 0 for the first hearteat. I just removed the `( )`

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**